.github/workflows/build.yml

```diff
@@ -250,6 +250,7 @@ jobs:
 
   incremental-mutants:
     runs-on: ubuntu-latest
+    if: github.ref_name != 'main' # `main` has no diff with itself
```

Comment:
Yeah it makes the expression a bit more complex and I had considered it. Or it requires an extra step before. I'll see how it can fit in this expression.